src/pg/sql/41_observatory_augmentation.sql

```diff
@@ -760,16 +760,13 @@ BEGIN
               || ') val_' || colid, ', ')
             || '
             FROM _procgeoms_' || Coalesce(geom_tablename || '_' || geom_geomref_colname, api_method) || ' _procgeoms ' ||
-              Coalesce(', ' || String_Agg(DISTINCT
-                  Coalesce('observatory.' || numer_tablename,
+              Coalesce(String_Agg(DISTINCT
```

Comment:
I think that this query is incorrect if it fall backs to the `LATERAL`.

It will generate something like
`FROM _proc_geoms_colname LATERAL (SELECT...`

Shouldn't here be a comma before `LATERAL` (as it is listing it as a new table expression, instead of as a join)?

I don't understand very well what this is doing here, it seems like a way to pass functions instead of tables to `OBS_GetData` which is quite confusing to me.